include/util.h

```diff
@@ -29,6 +29,28 @@ namespace connect
     std::string toString(const Events& events);
     std::string toString(const ObjectStream& objectStream);
 
+    inline Status makeSuccess(int code = Status::SUCCESS, int facility = Status::FACILITY_NONE)
```

Comment:
Again, Status is for SDK use. SDK users they aren't supposed to create values, but to check them.
And yes, it was initially together with Status class in domain-types.h header.